bundles/org.openhab.binding.solarmax/README.md

```diff
@@ -0,0 +1,78 @@
+# SolarMax Binding
+
+This binding supports SolarMax PV inverters.
+
+## Supported Things
+
+The SolarMax MT Series is support. (tested with 8MT2 devices)
+
+## Discovery
+
+Auto-discovery is currently not available.
+
+## Thing Configuration
+
+The IP address and port number (default 12345) of the device needs to be configured.
+
+
+```
+############################## openHAB SolarMax Binding #############################
+
+# The IP address or hostname of the SolarMax device
+#host=192.168.1.151|SolarMax1
+
+# The port number configured on the SolarMax device
+# Default is 12345
+#portNumber=12345
+
+# The refresh interval (in seconds) 
+# Default is 15 
+#refreshInterval=15
+```
+
+## Channels
+
+| channel                  | type  | description                                            |
+| ------------------------ | ----- | ------------------------------------------------------ |
+| LastUpdated              | Point | When was the data last read from the device            |
```

Comment:
```suggestion
| lastUpdated              | DateTime | Time when data was last read from the device.            |
```
Please use the correct channel ID from the thing-type.xml file so that people who use textual config get the 100% correct channel name to use. Note it is not a point, but the type of channel that you have written in the already mentioned xml file. Please do for the rest of this list.